python/ray/scripts/scripts.py

```diff
@@ -567,6 +567,9 @@ def get_head_ip(cluster_config_file, cluster_name):
 
 
 def main():
+    logging.basicConfig(
```

Comment:
@mitar I trust this the right way to initialize logging?

Reply:
Yes.